src/data/ACTIONS/root/SCH.ts

```diff
@@ -233,7 +243,22 @@ export const SCH = ensureActions({
 		name: 'Chain Stratagem',
 		icon: iconUrl(2815),
 		cooldown: 120000,
-		statusesApplied: ['CHAIN_STRATAGEM'],
+		statusesApplied: ['CHAIN_STRATAGEM', 'IMPACT_IMMINENT'],
+		combo: {
+			start: true,
+		},
+	},
+
+	// Baneful Impaction
+	BANEFUL_IMPACTION: {
+		id: 37012,
+		name: 'Baneful Impaction',
+		icon: iconUrl(2879),
+		cooldown: 300,
+		combo: {
+			from: 7436,
+		},
+		statusesApplied: ['BANEFUL_IMPACTION'],
```

Comment:
Baneful Impaction isn't a combo skill - you don't lose the ability to cast it just because you do something else in between.